src/form-question-types.rst

```diff
@@ -1681,7 +1681,11 @@ type
 appearance
   :tc:`none`
 
-Records audio using the device's microphone or a connected external microphone. By default, an :ref:`external application <external-audio-app>` is used. Starting with Collect v1.29, you can also use :ref:`built-in recording <built-in-audio-recording>`.
+Records audio using the device's microphone or a connected external microphone. By default, an :ref:`external application <external-audio-app>` is used. 
+
+.. tip::
+
+  Instead of the default audio widget, we recommend you use the :ref:`built-in audio recorder <built-in-audio-recording>` because you can customize audio quality and record while filling out other questions or using another app. Built-in recording is available in Collect v1.29 or later.
```

Comment:
Just as a side note: this messes with my mental model a bit. I'd think of there being one audio widget that can behave in different ways, rather than there being multiple audio widgets. It does look like this way of describing it is consistent with the geo widgets though, so I think that's on me.

Reply:
I think we can drop the "or using another app" here. That'd be possible with an external recorder as well.

Reply:
Agree that widgets are a weird term! We were young and foolish back then. File an issue with your suggested terminology. Field types? Entry elements? ODK experiences?